templates.json

```diff
@@ -0,0 +1,77 @@
+[ 
+    {
+
+        "path": "iOSNativeSwiftTemplate",
+        "description": "Basic Swift application",
```

Comment:
@rwhitley Could you review the descriptions? They will be shown to the use when using the new forceios/forcedroid/forcereact/forcehybrid listtemplates (or sfdx mobilesdk:ios:listtemplates).